### ISSUE TITLE
LiteRT runtime build not working for Linux+Aarch64

### ISSUE THREAD

**Issue by mansnils** [google-ai-edge/LiteRT#5]
There is no aarch64 linux wheel for ai-edge-litert.
https://pypi.org/project/ai-edge-litert/#files

For tensorflow there is e.g. CPython 3.11 manylinux: glibc 2.17+ ARM64
https://pypi.org/project/tensorflow/#files

It appears not to be possible to build natively for aarch64+linux from source (using the ci/build_pip_package_with_bazel.sh script): 

How to reproduce on Linux+Aarch64:
./ci/build_pip_package_with_bazel.sh 

Building TFL runtime in tensorflow on Linux+Aarch64 works:
./tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh

**Comment by byarbrough** [comment]
Bummed that I cannot install on the Pi 4 running `Linux raspberrypi 6.6.51+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.51-1+rpt3 (2024-10-08) aarch64 GNU/Linux`

*But* the docs do still say to use 

```
pip install tflite-runtime
```
https://ai.google.dev/edge/litert/microcontrollers/python

**Comment by AllanKamimura** [comment]
yeah..... it's really curious how a library aimed for edge-runtime don't have an arm64 build available
I asked if they have a date planned for the release of the linux/arm64 build, but didn't get any answer

in the end, I build it myself using that ci script and it is working :upside_down_face: 
python 3.11
linux/arm64 

[ai_edge_litert_nightly-1.0.1.dev20241024-cp311-cp311-linux_aarch64.zip](https://github.com/user-attachments/files/17514029/ai_edge_litert_nightly-1.0.1.dev20241024-cp311-cp311-linux_aarch64.zip)

well, use it at your own discretion :sweat_smile:

**Comment by johnthagen** [comment]
`aarch64` wheels are needed to make `ai-edge-litert` a true replacement for `tflite-runtime`, which provides `manylinux_2_34_aarch64.whl` wheels.

- https://pypi.org/project/tflite-runtime/2.14.0/#files

Another example where this is needed is for macOS ARM64 local development within containers-- this requires an ARM64 Linux wheel when installing into the container.

**Comment by AndreasX21** [comment]
Ran into the same issue, we need linux/arm64 wheels asap!

**Comment by johnthagen** [comment]
What's a bit disconcerting is that on the `-nightly` PyPI releases, _all_ Linux wheels are missing. Only macOS ARM wheels are present 😕 

- https://pypi.org/project/ai-edge-litert-nightly/1.0.1.dev20250105/#files

**Comment by johnthagen** [comment]
Perhaps related

- #350

**Comment by johnthagen** [comment]
On the latest `ai-edge-litert-nightly` both x86 and ARM64 wheels appear for Linux 😱 🚀 

- https://pypi.org/project/ai-edge-litert-nightly/1.0.1.dev20250112/#files

**Comment by byarbrough** [comment]
> On the latest `ai-edge-litert-nightly` both x86 and ARM64 wheels appear for Linux 😱 🚀
> 
>     * https://pypi.org/project/ai-edge-litert-nightly/1.0.1.dev20250112/#files

Confirmed you can install and use the -nightly on a Pi :)

**Comment by johnthagen** [comment]
As of `ai-edge-litert` 1.1.0, ARM64 wheels for Linux are now published

- https://pypi.org/project/ai-edge-litert/1.1.0/#files

I think this issue can be closed 🚀 .

**Comment by johnthagen** [comment]
Though, hopefully I didn't speak too soon as now the latest _nightly_ is missing Linux ARM wheels again:

- https://pypi.org/project/ai-edge-litert-nightly/1.1.2.dev20250212/#files

I hope this is a temporary CI issue.

**Comment by pkgoogle** [comment]
Hi All, @johnthagen, @mansnils This should be fixed now: https://pypi.org/project/ai-edge-litert/#files, please feel free to close if there is no more open issues.
